packages/react/src/hooks/useFederatedModuleContainer.ts

```diff
@@ -52,24 +52,31 @@ export const useFederatedModuleContainer = (
                     scriptTag.src = sanitizedUrl;
                     scriptTag.async = true;
                     scriptTag.onload = async () => {
-                        container = <RemoteContainer | undefined>(
+                        localContainer = <RemoteContainer | undefined>(
                             window[entry as keyof Window]
                         );
-                        if (!container) {
+                        if (!localContainer) {
                             console.warn(
                                 "container still undefined after federated module import"
                             );
                             return;
                         }
-                        await container.init(shareScope);
-                        container.__initialized = true;
-                        if (!cancelled) setContainer(container);
+                        try {
```

Comment:
I'm not sure if this is the case but this `try/catch` may be "nullifying" the error boundaries around the `TemplateComponent`? Since we are catching the error here, without rethrowing it.

Reply:
It shouldn't influence it because we're a step before that. Here we're just initializing the federated module container from which we'll take the React components to use. We'll actually use the component later and the error boundary is applied then.